src/custom_request_schedule_manager.cc

```diff
@@ -52,62 +61,35 @@ CustomRequestScheduleManager::CustomRequestScheduleManager(
           params.batch_size, params.measurement_window_ms, params.max_trials,
           params.max_threads, params.num_of_sequences,
           params.shared_memory_type, params.output_shm_size,
-          params.serial_sequences, parser, factory, params.request_parameters)
+          params.serial_sequences, parser, factory, params.request_parameters),
+      warmup_request_count_(params.warmup_request_count),
+      benchmark_request_count_(params.request_count)
 {
-  max_threads_ = std::min(max_threads_, params.request_count);
```

Comment:
Since there are two phases of the fixed schedule mode (CustomRequestScheduleManager), `max_threads_` needs to be potentially altered before each begin.

This potential altering happens [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#diff-208ed87aa8e88c62da2501d92468c47876fd526bd164a71fd4d9d1b4536194dbR162) for warmup and [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#diff-208ed87aa8e88c62da2501d92468c47876fd526bd164a71fd4d9d1b4536194dbR209) for the following benchmark